content/pt-br/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+O CRI é uma interface de plug-in que permite ao kubelet usar uma ampla variedade
+ de tempos de execução de contêiner, sem a necessidade de recompilar 
+ os componentes do cluster.
+
+Você precisa de um
+{{<glossary_tooltip text="tempo de execução de contêiner" term_id="container-runtime">}} em funcionamento em cada nó em seu cluster,
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} possa iniciar
+{{< glossary_tooltip text="Pods" term_id="pod" >}} e seus contêineres.
+
+{{< glossary_definition prepend="The Container Runtime Interface (CRI) is" term_id="container-runtime-interface" length="all" >}}
+
+<!-- body -->
+
+## A API {#api}
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+O kubelet atua como um cliente ao se conectar ao tempo de execução do contêiner 
+via GRPC. Os endpoints de serviço de imagem e tempo de execução devem estar 
+disponíveis no tempo de execução do contêiner, que pode ser configurado 
+paradamente no kubelet usando os sinalizadores de linha de comando 
```

Comment:
Some bits in this sentence:
* Typo in "paradamente".
* Perhaps "opções de linha de comando" would sound better, along with the link?